src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -128,7 +131,7 @@ public static void renderPlayerArm(RenderArmEvent event) {
             }
         }
 
-        if (event.getArm() == HumanoidArm.LEFT) {
+        else if (event.getArm() == HumanoidArm.LEFT) {
```

Comment:
The event is fired twice - once for left arm and once for right arm.

It does not get both arms simultaneously, so you can avoid checking the left arm if you know it's the right arm.